miner/algo_common.go

```diff
@@ -301,7 +329,16 @@ func (envDiff *environmentDiff) commitBundle(bundle *types.SimulatedBundle, chDa
 
 	bundleProfit := coinbaseBalanceDelta
 
-	bundleActualEffGP := bundleProfit.Div(bundleProfit, big.NewInt(int64(gasUsed)))
+	var (
+		gasUsedBigInt     = new(big.Int).SetUint64(gasUsed)
+		bundleActualEffGP *big.Int
+	)
+	if gasUsed == 0 {
```

Comment:
Can this ever be true? I find this a bit weird. Maybe you should use an assert instead if this should never be the case but you don't want to do the division without checking.

Reply:
yes, unit test was failing without this line

Reply:
it's not failing anymore, i think because logic changed, updated to no longer check for 0 since tests are passing